5.sc_Morphology_Logistic_Regression/1.train_models/0.train_single_class.ipynb

```diff
@@ -0,0 +1,577 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #15.        class_weight="balanced",
</pre><p>I think this should be ok, but in the case of severe class imbalance, it may not be sufficient to compensate for the class imbalance</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WayScience/pyroptosis_signature_data_analysis/pull/58/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>